### PR TITLE
feat: VSDSPUB-245: Update LdesConfig

### PR DESCRIPTION
**Pull request by WLefever-Cegeka**
To be merged after #169

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ These need to be added in the `application.yml` file in `ldes-server-application
       host-name: {endpoint of LDES Server}
       member-type: {Defines the which syntax type is used to define the member id}
       shape: {URI to defined shape}
+      timestamp-path: {SHACL property path to the timestamp when the version object entered the event stream.}
```
Comment (sverholen) [inline README.md:86]: RDF predicate ?

File: ldes-fragmentisers/ldes-fragmentisers-sequential/ldes-fragmentisers-sequential-impl/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/sequential/SequentialFragmentationServiceTest.java [hunk 2]
```diff
@@ -70,7 +70,7 @@ void when_MemberIsNotFoundInRepository_thenMemberNotFoundExceptionIsThrown() {
     void when_NoFragmentExists_thenFragmentIsCreatedAndMemberIsAdded() throws IOException {
         String ldesMemberString = FileUtils.readFileToString(ResourceUtils.getFile("classpath:example-ldes-member.nq"), StandardCharsets.UTF_8);
         LdesMember ldesMember = new LdesMember("https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10810464/1", RdfModelConverter.fromString(ldesMemberString, Lang.NQUADS));
-        LdesFragment createdFragment = new LdesFragment("fragmentId", new FragmentInfo("view", "shape", "viewShortName", List.of(new FragmentPair("Path", "Value"))));
+        LdesFragment createdFragment = new LdesFragment("fragmentId", new FragmentInfo("viewShortName", List.of(new FragmentPair("Path", "Value"))));
         when(ldesMemberRepository.getLdesMemberById(ldesMember.getLdesMemberId())).thenReturn(Optional.of(ldesMember));
         when(ldesFragmentRespository.retrieveOpenFragment(ldesConfig.getCollectionName()))
```
Comment (sverholen) [inline ldes-fragmentisers/ldes-fragmentisers-sequential/ldes-fragmentisers-sequential-impl/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/sequential/SequentialFragmentationServiceTest.java:75]: respository

File: ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/LdesConfigTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+@SpringBootTest(classes = {LdesConfig.class})
+@EnableConfigurationProperties
+@ActiveProfiles("test")
+class LdesConfigTest {
+
+    @Autowired
+    private LdesConfig ldesConfig;
+
+    @Test
+    @DisplayName("Verify contect of LdesConfig")
```
Comment (sverholen) [inline ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/LdesConfigTest.java:21]: contect

**Review by sverholen — APPROVED** [review]

File: ldes-fragmentisers/ldes-fragmentisers-sequential/ldes-fragmentisers-sequential-impl/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/sequential/SequentialFragmentationServiceTest.java (hunk 2 above)
Comment (WLefever-Cegeka) [reply ldes-fragmentisers/ldes-fragmentisers-sequential/ldes-fragmentisers-sequential-impl/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/sequential/SequentialFragmentationServiceTest.java]: fixed

File: ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/LdesConfigTest.java (hunk 3 above)
Comment (WLefever-Cegeka) [reply ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/LdesConfigTest.java]: fixed

**Pull request merged**